.github/workflows/test.yml

```diff
@@ -72,18 +72,18 @@ jobs:
     runs-on: ubuntu-latest
     needs: tests
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-python@v5
         with:
           # Use latest Python, so it understands all syntax.
           python-version: ${{env.PYTHON_LATEST}}
 
       - run: python -Im pip install --upgrade "coverage[toml]>=7.2,<8.0"
 
       - name: ⬇️ Download coverage data
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
```

Comment:
best to keep this and `upload-artifact` to v3, or follow the instructions from https://hynek.me/articles/ditch-codecov-python/

Reply:
Hi @zerolab ,
Thank you for the resource! I pushed forward with the version upgrade.
https://github.com/torchbox-forks/wagtail-footnotes/pull/2/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R66